src/index.html

```diff
@@ -9,9 +9,10 @@
 <body>
   <div class="container">
     <h1 class="title">My Daily Todo</h1>
-   <form action="#">
-    <input autofocus type="text" name="todo" id="new-todo" placeholder="Add new todo">
-   </form>
+    <form action="#" class="form">
+      <input type="text" name="todo" id="new-todo" placeholder="Add new todo">
+    </form>
+    <p class="feedback"></p>
     <ul class="todo-list">
     </ul>
     <p class="clear-all">Clear all completed</p>
```

Comment:
- [ ] Kindly this `p` should be a button with an `id` so you can use it to clean the array by deleting everything once.